backend/api/Profiles/BuildingProfile.cs

```diff
@@ -36,10 +36,13 @@ public BuildingProfile()
                 .ForMember(dest => dest.BuildingConstructionType, opt => opt.Ignore())
                 .ForMember(dest => dest.BuildingPredominateUse, opt => opt.Ignore())
                 .ForMember(dest => dest.AddressId, opt => opt.MapFrom(src => src.Address.Id))
+                .ForMember(dest => dest.Address, opt => opt.MapFrom(src => src.Address))
+                .ForMember(dest => dest.ParcelId, opt => opt.MapFrom(src => src.ParcelId))
                 .ForMember(dest => dest.Parcel, opt => opt.Ignore())
+                .ForMember(dest => dest.AgencyId, opt => opt.MapFrom(src => src.AgencyId))
                 .ForMember(dest => dest.Agency, opt => opt.Ignore())
-                .ForMember(dest => dest.IsSensitive, opt => opt.Ignore())
-                .ForMember(dest => dest.Evaluations, opt => opt.MapFrom(src => src.Evaluations))
+                .ForMember(dest => dest.IsSensitive, opt => opt.MapFrom(src => src.IsSensitive))
+                .ForMember(dest => dest.Evaluations, opt => opt.Ignore())
```

Comment:
I think this'll prevent my parcel update service call from saving building evaluations into the database.

Reply:
It does on purpose.  Automapper will delete all buildings first then add new ones.  Not the desired behaviour.

Reply:
I have updated the service layer so that it adds/updates/ buildings and their evaluations.